lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -9,7 +9,12 @@ def fn_name(): return sys._getframe(1).f_code.co_name
 try:
     import gi
 except ImportError:
-    raise ImportError("Gtk3 backend requires pygobject to be installed.")
+    try:
+        import pgi
+        pgi.install_as_gi()
```

Comment:
I know what you mean, I think I would prefer:
```python
import pgi as gi
```
Possibly creating a ``gtk3_compat.py`` file as we do with Qt...